qi-doc/scribblings/list-operations.scrbl

```diff
@@ -0,0 +1,76 @@
+#lang scribble/doc
```

Comment:
RE 474b062d78d87816496f805f6289168818362f43, Binding fix belongs in the commit that introduced the problem. The scribblings
could be an independent commit or part of the previous commit that introduces
the bindings.